dvc/remote/base.py

```diff
@@ -962,7 +962,7 @@ def checkout(
         skip = False
         if not checksum:
             logger.warning(
-                "No checksum info found for '{}'. " "It won't be created.",
+                "No file hash info found for '{}'. " "It won't be created.",
```

Comment:
Again, we call it checksum everywhere else, it is weird that we do this change in this PR.

Reply:
Sorry just noticed this and it's been merged with the change at this point. I would've rolled these back if I had the capacity at the time. In any case they have all been changed in a separate PR on top of this one now. Resolving 🙂